### ISSUE TITLE
Workflow IDs containing / character do not load

### ISSUE THREAD

**Issue by jlegrone** [temporalio/ui#668]
**Describe the bug**
Workflow IDs containing `/` characters cause the web UI to display a 404 error.

**To Reproduce**

1. Install Temporalite
2. `temporalite start --ephemeral --namespace default`
3. In another terminal window: `tctl wf start --tq repro_slash_bug --workflow_type repro_slash_bug -w 'foo/bar'`
4. Open http://localhost:8233/namespaces/default/workflows?search=basic&query=WorkflowType%3D%22repro_slash_bug%22
5. Click on `foo/bar` workflow execution and observe 404

**Expected behavior**
Any valid workflow ID should load in the Temporal web UI.

**Screenshots**

<img width="1440" alt="Screen Shot 2022-07-06 at 16 57 57" src="https://user-images.githubusercontent.com/6752382/177642670-8d243de7-4db3-49c4-97c7-83a9f96ba021.png">

**Desktop (please complete the following information):**

- OS: macOS
- Browser: Firefox
- Version: 102.0

**Additional context**

This could potentially be classified as a ui-server bug instead. Eg. the following works to reproduce as well:

```
$ curl 'http://localhost:8233/api/v1/namespaces/default/workflows/foo%252Fbar/runs/3e93502b-4e66-4f13-865c-0d60236a89a5?'
{
  "code": 5,
  "message": "Workflow executionsRow not found.  WorkflowId: foo%2Fbar, RunId: 3e93502b-4e66-4f13-865c-0d60236a89a5",
  "details": [
    {
      "typeUrl": "type.googleapis.com/temporal.api.errordetails.v1.NotFoundFailure",
      "value": null
    }
  ]
}
```

**Comment by stevekinney** [comment]
Do you know which version `ui` and `ui-server` you're using in Temporalite? I'm trying to see if this is something we already addressed in #394 or an edge case that we missed.

**Comment by jlegrone** [comment]
Ah, looks like Temporalite is still on 0.13.1, which was released before that PR was merged. I'll close this. Thanks!

https://github.com/DataDog/temporalite/blob/80ecd27ddfcae2c1e60c5803bc516c26b27dad13/go.mod#L8
